metadata/src/main/java/org/apache/kafka/controller/metrics/ControllerMetadataMetricsPublisher.java

```diff
@@ -115,9 +115,6 @@ private void publishDelta(MetadataDelta delta) {
             }
```

Comment:
We still have ZK_MIGRATION_STATE and MIGRATING_ZK_BROKER_COUNT defined in ControllerMetadataMetrics. Should we delete these too?